docs/source/using/basic-usage.adoc

```diff
@@ -177,9 +177,12 @@ This environment variable is currently experimental and semantics might change i
 === Persistent Configuration
 
 Using persistent configuration allows you to control {project} behavior without specifying actual command line flags, similar to the way you use environment variables.
+You can specify persistent configuration as global (using `global` flag) to make each profile consumes it without setting it for individual profile.
 
 {project} maintains a configuration file in *_$MINISHIFT_HOME/config/config.json_*.
-This file can be used to set commonly-used command line flags persistently.
+This file can be used to set commonly-used command line flags persistently for individual profile.
+{project} also maintain global configuration file in *_$MINISHIFT_HOME/config/global.json_*.
```

Comment:
`The global configuration file is maintained at *_$MINISHIFT_HOME/config/global.json_*`.